licenses/models.py

```diff
@@ -342,4 +342,30 @@ class TranslationBranch(models.Model):
     last_transifex_update = models.DateTimeField(
         "Time when last updated on Transifex.", null=True, blank=True,
     )
-    complete = models.BooleanField("Only one incomplete per branch", default=False,)
+    complete = models.BooleanField(default=False)
+
+    class Meta:
+        verbose_name_plural = "translation branches"
+
+    def __str__(self):
+        return f"Translation branch {self.branch_name}. {'Complete' if self.complete else 'In progress'}."
+
+    @property
+    def stats(self):
+        untranslated_messages = 0
+        translated_messages = 0
+        for code in self.legalcodes.all():
+            pofile = code.get_pofile()
+            untranslated_messages += len(pofile.untranslated_entries())
+            translated_messages += len(pofile.translated_entries())
```

Comment:
can we prefix these variables with `num_of`

Reply:
sure.